client/src/components/Login.style.js

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+  background-color: var(--main-color, #f9d52f);
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-flow: column wrap;
+`;
+
+export const H1 = styled.h1`
+  font-style: var(--black);
+  background-color: var(--white);
+  margin: 50px;
+`;
+export const Form = styled.form`
```

Comment:
sorry, this was probably my doing, but since this isn't doing anything, we can remove this style. that also means you'll have to change the html to ```form``` instead of ```S.Form```

Reply:
Did you mean to put `html` instead of `S.Form`